### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by carpii**
Attempt to document more options (not complete) 
Added FAQ and Troubleshooting section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -286,6 +300,49 @@ node node_modules/grunt-cli/bin/grunt
...
+Include jQuery, then EmojiOne, then EmojiOneArea scripts
+
+#### Can I use EmojiOneArea to just display Emoji icons in a div?
+
+EmojiOneArea is intended to be a text editor which supports EmojiOne. 
+If you just want to display Emoji icons, the EmojiOne library already provides everything you need. 
+
+#### Can I add extra buttons into EmojiOneArea, alongside the existing emoji picker icon?
+
+This is not currently supported, but is being looked at (see
```
Comment (anamauad) [inline README.md]: Some information is missing here.

File: README.md [hunk 2]
```diff
@@ -286,6 +300,49 @@ node node_modules/grunt-cli/bin/grunt
...
+
+You can use the pickerPosition option which allows you to control where the picker appears in relation to the source input.
+For more control, you could use translate CSS
+
+## Known Issues
+
+#### Internet Explorer focus issues
+IE 11 causes EmojiOneArea to hide (and trigger blur event) when the emoji picker scrollbar is clicked (See #127) 
+
+#### EmojiOneAea positioning
```
Comment (anamauad) [inline README.md]: There's a misspelling here: needs correction to EmojiOneArea

File: README.md [hunk 3]
```diff
@@ -286,6 +300,49 @@ node node_modules/grunt-cli/bin/grunt
...
+EmojiOneArea is intended to be a text editor which supports EmojiOne. 
+If you just want to display Emoji icons, the EmojiOne library already provides everything you need. 
+
+#### Can I add extra buttons into EmojiOneArea, alongside the existing emoji picker icon?
+
+This is not currently supported, but is being looked at (see 
+
+#### Firefox is not positioning the input caret correctly in EmojiOneArea
+
+This appears to be a long standing FireFox bug, related to contenteditable, the placeholder attribute, and the psuedo :before or :after classes
```
Comment (anamauad) [inline README.md]: There's a misspelling here: needs correction to "pseudo"

**Comment by carpii** [comment]
Thanks @anamauad 

@mervick Should I fix these as a 2nd commit, or just fix and then force-push the docs branch?

**Comment by mervick** [comment]
@carpii you can add additional commits

**Comment by carpii** [comment]
Should there be separate documentation for 2.1 vs 3.x ?

**Comment by mervick** [comment]
Yes, I think we can separate it 2 different README files

**Comment by mervick** [comment]
You can skip 2.x docs at all

**Comment by carpii** [comment]
Ok! I will split them out into diff docs

**Pull request merged**